inputTypes/autocomplete/autocomplete.js

```diff
@@ -121,6 +122,11 @@ Template.afAutocomplete.onRendered(function () {
     }
   })
 
+  const updateValue = value => {
+    $hidden.val(value)
+    $hidden.trigger('change')
+  }
+
```

Comment:
Clever! Nicely done. Is it safe to assume that the change event triggers validation? on blur too?

Reply:
I see the blur change below.

Reply:
The change is used to fire Updates for any reactive getter like AutoForm.getFieldValue

While blur triggers validation.